tasks/rbd_fio.py

```diff
@@ -183,16 +183,17 @@ def run_fio(remote, config):
     try:
         log.info("Running rbd feature - fio test on {sn}".format(sn=sn))
         fio = "https://github.com/axboe/fio/archive/fio-" + fio_version + ".tar.gz"
-        remote.run(args=['mkdir', run.Raw('~/rbd-fio-test'),])
-        remote.run(args=['cd' , run.Raw('~/rbd-fio-test'),
+        rbd_test_dir = "~/cephtest/rbd-fio-test"  #let nuke cleanup in case of abrupt failures
```

Comment:
Instead of hardcoding this, I would use this function:
https://github.com/ceph/teuthology/blob/master/teuthology/misc.py#L132

Just to avoid potential weirdness if that changes someday.